### PR TITLE
removed outdated sections from home page, added logos to ruby jobs sections, changed css of buttons

### PR DESCRIPTION
**Pull request by chewonithard**
- still unable to get telegram logo to show, help would be appreciated
- same for heart logo at footer
- one of the company logo is not showing, will fix it

### CODE REVIEW

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -32,22 +32,19 @@
...
-  $padding: 10rem;
+  padding: 3rem 0;
 
-  padding-bottom: $padding;
-  padding-top: $padding;
-
-  &.section-gray {
-    background-color: $theme-gray;
+  &.section-blue {
+    background-color: #bfe3ff;
```
Comment (TayKangSheng) [inline app/assets/stylesheets/base.scss:38]: can extract it into a color variable inside variables.scss

File: app/assets/stylesheets/base.scss [hunk 2]
```diff
@@ -168,17 +128,20 @@ footer {
...
   .section {
     padding-left: 4rem;
     padding-right: 4rem;
   }
 }
 
+#chat-with-us {
+  color: #222529;
+  .section-heading {
+    color: #222529;
```
Comment (TayKangSheng) [inline app/assets/stylesheets/base.scss:141]: same comment on extracting color as variable

**Review by TayKangSheng — COMMENTED** [review]
Generally looks okay to me. Will test it and check back again.

**Review by TayKangSheng — COMMENTED** [review]
Need to fix a few more things. Otherwise it looks good. Let me know once you've put in the changes and we can merge the code 🚀

File: app/assets/stylesheets/base.scss (hunk 1 above)
Comment (chewonithard) [reply app/assets/stylesheets/base.scss]: done

File: app/assets/stylesheets/base.scss (hunk 2 above)
Comment (chewonithard) [reply app/assets/stylesheets/base.scss]: done

File: app/views/application/index.html.slim [hunk 3]
```diff
@@ -1,74 +1,37 @@
...
+        and share 5 random Ruby tips. Building & shipping is how we roll.
+        It's a great way to meet Ruby enthusiasts!
+      a.btn.btn-lg.btn-primary href='http://www.meetup.com/Singapore-Ruby-Group' target='_blank'
+        span Come Join Us!
 
-.section#chat-with-us
+.section.section-blue#chat-with-us
   .container
-    h3.section-heading.heading-line Chat With Us
+    h3.section-heading Chat with us on <i class="fa-brands fa-telegram"></i>
```
[the remark this answers is not in the document]
Comment (chewonithard) [reply app/views/application/index.html.slim]: done thanks

File: app/views/application/index.html.slim [hunk 4]
```diff
@@ -1,74 +1,37 @@
...
     .row.text-center
-      .col-sm-8.col-sm-offset-2
-        = rubysg_telegram_qr_code_svg
-
-        p
-          strong Invite yourself to our <a href="https://t.me/joinchat/TWJ3HtLeJF81EweA" target="_blank">Telegram group</a>
+      a.btn.btn-lg.btn-primary href='https://t.me/joinchat/TWJ3HtLeJF81EweA" target="_blank">Telegram group</a>' target='_blank'
```
[the remark this answers is not in the document]
Comment (chewonithard) [reply app/views/application/index.html.slim]: done

**Review by TayKangSheng — APPROVED** [review]
looks good. but there is conflict with master. got to fix it before this PR can be merged.

**Comment by chewonithard** [comment]
I think i fixed the conflict and merged with master (not sure if that's the correct thing to do)

**Comment by TayKangSheng** [comment]
Thanks for the fix! Merging now.

**Pull request merged**